bin/Debugger.Sample/Debugger.Sample.cpp

```diff
@@ -131,15 +163,32 @@ JsErrorCode RunScript(const wchar_t* filename, JsValueRef* result)
         return JsErrorInvalidArgument;
     }
 
-    // Load the script from the disk.
-    std::wstring script = LoadScript(fullPath);
-    if (script.empty())
+    if (loadScriptUsingBuffer)
     {
-        return JsErrorInvalidArgument;
+        JsValueRef sourceUrl;
+        JsValueRef scriptValueRef;
+
+        // Load script from the file
+        std::string script = LoadScriptUtf8(fullPath);
+        if (script.empty())
+        {
+            return JsErrorInvalidArgument;
+        }
+
+        IfFailRet(JsPointerToString(fullPath, wcslen(fullPath), &sourceUrl));
+        IfFailRet(JsCreateExternalArrayBuffer(const_cast<char*>(script.c_str()), static_cast<unsigned int>(script.length()), NULL, NULL, &scriptValueRef));
```

Comment:
We still have the lifetime issue from my previous comment here.  We need to ensure that the data returned by `c_str` is valid for the entire lifetime of the external array buffer.

Reply:
Yes, I pushed other changes but haven't had a chance to address this yet.

Reply:
Fixed. Please see the latest commits.